docs/api-reference/carto/carto-layer.md

```diff
@@ -111,6 +111,14 @@ Required when apiVersion is `API_VERSIONS.V3`.
 
 Name of the connection registered in the CARTO workspace.
 
+##### `geoColumn` (String, optional)
```

Comment:
We need to say that this parameter is only supported for `MAP_TYPES.TABLE`, so only available in v3 because type table is only supported in V3+